radixdlt/src/integration/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
This is copied over from https://github.com/radixdlt/radixdlt-core/blob/release/1.0-beta.7/radixdlt/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

Fixes failing integrationTest due to Mockito not being able to mock final class (`AID.java`)